ui/app/registration/controllers/searchPatientController.js

```diff
@@ -5,8 +5,12 @@ angular.module('bahmni.registration')
         'messagingService', '$translate', '$filter',
         function ($rootScope, $scope, $location, $window, spinner, patientService, appService, messagingService, $translate, $filter) {
             $scope.results = [];
+            var searchDisplayOptions = appService.getAppDescriptor().getConfigValue("patientSearchResultOptions") || {};
+            var ignoredIdentifiers = new Set(searchDisplayOptions.ignorePatientIdentifiers || []);
+            $scope.showAge = searchDisplayOptions.showAge != null ? searchDisplayOptions.showAge : true;
+            $scope.showDOB = searchDisplayOptions.showDOB != null ? searchDisplayOptions.showDOB : false;
```

Comment:
instead of this way, can we just set it like `$scope.showAge = searchDisplayOptions.showAge` ? can you confirm if that works

Reply:
We also need a default behaviour Kavitha, incase the config is not present. we should then display only age